src/Console/Locale/CopyLocaleCommand.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Console\Locale;
+
+use Backend\Core\Language\Locale;
+use Common\Locale as CommonLocale;
+use Common\ModulesSettings;
+use ForkCMS\Utility\Module\CopyContentToOtherLocale\CopyContentFromModulesToOtherLocaleManager;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * This command will copy all the content from all the modules
+ */
+class CopyLocaleCommand extends Command
+{
+    /**
+     * @var CopyContentFromModulesToOtherLocaleManager
+     */
+    private $copyContentFromModulesToOtherLocaleManager;
+
+    /**
+     * @var SymfonyStyle
+     */
+    private $formatter;
+
+    /**
+     * @var ModulesSettings
+     */
+    private $settings;
+
+    public function __construct(
+        ModulesSettings $settings,
+        CopyContentFromModulesToOtherLocaleManager $copyContentFromModulesToOtherLocaleManager
+    ) {
+        parent::__construct();
+
+        $this->settings = $settings;
+        $this->copyContentFromModulesToOtherLocaleManager = $copyContentFromModulesToOtherLocaleManager;
+    }
+
+    protected function configure(): void
+    {
+        $this->setName('forkcms:locale:copy')
```

Comment:
I would use a different name for the command, if I read this I expect that I'm copying translations or something like that

Reply:
maybe `forkcms:pages:copy-to-locale` ?

Reply:
That's a beter name. Changed 95b5e67f